maintainers/maintainer-list.nix

```diff
@@ -10548,6 +10548,13 @@
     githubId = 7221768;
     name = "Andika Demas Riyandi";
   };
+  rkjnsn = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
I added myself as maintainer instead, @rkjnsn seems to have lost interest.